adr/2022_05_09_mobile_click_behavior.md

```diff
@@ -0,0 +1,30 @@
+# Mobile click behavior
+
+> Status: proposed
+
+> Date: 09.05.2022
+
+## Context
+
+Currently, a short click will switch the app in fullscreen mode. A long click (with a release of the click) will run an identify (like a click on desktop)
+
+The issue is that there is no location popup possibilities with that (or at least it's unwanted if it pops up after a long click). We need to rework that.
+
+# Decision
+
+Switch the short click to the identify (as on desktop). A long click should show the location popup, and if possible a long touch should be detected without the user needing to raise his finger (no release detection). The threshold is 500ms.
+
+To replace the fullscreen toggle, I would propose we add a new button in the interface (maybe by the zoom/geolocation?) with the "expand" icon from font-awesome.
```

Comment:
What about using the double click for fullscreen toggle ? Currently double click is for zoom in, but zoom can also be done with the scrool mouse or two finger move.